gffquant/annotation/genecount_annotator.py

```
@@ -0,0 +1,62 @@
+        # self.total_gene_counts, u_sf, a_sf = counter.generate_gene_count_matrix(refmgr)
+        # logger.info("TOTAL_GENE_COUNTS = %s", self.total_gene_counts)
+
+        # writer.write_gene_counts(
+        #     counter,
+        #     refmgr,
+        #     u_sf, a_sf,
+        #     gene_group_db=gene_group_db,
+        # )
+
+        # formerly used in compute_count_vector
+        strand_specific_counts = (
```

Ruff (0.8.2)

Local variable `strand_specific_counts` is assigned to but never used

Remove assignment to unused variable `strand_specific_counts`

(F841)